packages/battery/battery_platform_interface/lib/enums/battery_state.dart

```diff
@@ -7,5 +7,8 @@ enum BatteryState {
   charging,
 
   /// The battery is currently losing energy.
-  discharging
+  discharging,
+
+  /// The battery is currently not charging(Android Only).
```

Comment:
nit: space between `charging (Android only)`.

We should probably make some attempt to explain a bit more about the difference between discharging and not charging.

Reply:
@amirh do we consider adding an enum value a breaking change or not?

Reply:
@dnfield wdyt ?

[isCharging](https://developer.android.com/reference/android/os/BatteryManager#isCharging())

BATTERY_STATUS_NOT_CHARGING : The device is plugged in and the battery level doesn't going up.